spec.md

```diff
@@ -74,13 +109,44 @@ can be found under the "omero" key in the group-level metadata:
 }
 ```
 
-
 See https://docs.openmicroscopy.org/omero/5.6.1/developers/Web/WebGateway.html#imgdata
 for more information.
 
+### "masks"
+
+The special group "masks" found under an image Zarr contains the key `masks` containing
+the paths to mask objects which can be found underneath the group:
+
+```
+{
+  "masks": [
+    "orphaned/0"
+  ]
+}
+```
+
+Unlisted groups MAY be masks.
+
+### "color"
+
+The `color` key defines an image that is "labeled", i.e. every unique value in the image
+represents a unique, non-overlapping object within the image. The value associated with
+the `color` key is another JSON object in which the key is the pixel value of the image and
+the value is an RGBA color (4 byte, `0-255` per channel) for representing the object:
+
+```
+{
+  "color": {
+    "1": 8388736,
+    ...
+```
+
```

Comment:
### "image"

The `image` key is an optional dictionary which contains information on the image the mask is associated with.
If included it must include a key `array` whose value that is either:
- A relative path to a Zarr image array, for example:
    ```
    {
      "image": {
        "array": "../../0"
      }
    }
    ```
- A URL to a Zarr image array (use this if the mask is stored seperately from the image Zarr), for example:
    ```
    {
      "image": {
        "array": "https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr/0"
      }
    }
    ```

Reply:
See also open discussion at https://github.com/ome/omero-cli-zarr/pull/19#issuecomment-663022544 about the specification of the `image` key